parachains/runtimes/assets/statemine/src/xcm_config.rs

```diff
@@ -66,6 +69,9 @@ pub type LocationToAccountId = (
 	SiblingParachainConvertsVia<Sibling, AccountId>,
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
+	// Different global consensus parachain sovereign account.
+	// (Used for over-bridge transfers and reserve processing)
+	GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>,
```

Comment:
We realise we need to something similar for Snowbridge. A converter that takes `/GlobalConsensus(Ethereum)/AccountKey20` and converts it into a local account id.

Lets chat more about in our sync meeting later today.